lib/core/src/server/cli/prod.js

```diff
@@ -13,10 +13,12 @@ function getCLI(packageJson) {
     .option('-c, --config-dir [dir-name]', 'Directory where to load Storybook configurations from')
     .option('-w, --watch', 'Enable watch mode')
     .option('--quiet', 'Suppress verbose build output')
+    .option('--loglevel [level]', 'Control level of logging during build')
     .option('--no-dll', 'Do not use dll reference')
     .option('--debug-webpack', 'Display final webpack configurations for debugging purposes')
     .parse(process.argv);
 
+  logger.setLevel(program.loglevel);
```

Comment:
What's the level being set to now, when the cli argument isn't given?

Would wrapping this with an if-statement make sense?

Reply:
I set a default value in the node-logger index.ts to 'info' to avoid an if wrapper.

Previously, no default was set, but node-logger doesn't wrap beyond 'info', so it's a safe value to support current functionality while supporting long-term best-practice of ensuring debugging output is protected from users who don't explicitly ask for it.